warehouse/utils/wheel.py

```diff
@@ -40,11 +40,25 @@
         re.compile(r"^macosx_(\d+)_(\d+)_(.*?)$"),
         lambda m: f"macOS {m.group(1)}.{m.group(2)}+ {_normalize_arch(m.group(3))}",
     ),
+    (
+        re.compile(r"^android_(\d+)_(.*?)$"),
+        lambda m: f"Android SDK {m.group(1)}+ {_normalize_arch(m.group(2))}",
```

Comment:
As mentioned [here](https://github.com/pypa/packaging.python.org/pull/1804#discussion_r1943019546), I'd prefer to replace "SDK" with "API level".